bootstrap/src/p2p/event_loop.rs

```diff
@@ -281,4 +304,12 @@ impl EventLoop {
 			},
 		}
 	}
+
+	fn handle_periodic_bootstraps(&mut self) {
+		// periodic bootstraps should only start after the initial one is done
+		if self.bootstrap.is_startup_done {
```

Comment:
This check will fail 90% of the time on larger network sizes due to the bootstrap query timeout.